frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -108,35 +108,71 @@ <h5 class="fw-light">Challenge Settings</h5>
-            >
-            Phase Details
-            <i class="fa fa-pencil" aria-hidden="true"> </i>
-            </button>
           </div>
 
         <div class="row margin-bottom-cancel phase-card " *ngIf="selectedPhase">
-              <span class="fs-16 fw-light">
-                <strong class="text-light-black">Is Public </strong>
-              </span>
-              <a class="pointer fs-20" (click)="togglePhaseVisibility()">
-                <i class="{{ phaseVisibility.icon }}" aria-hidden="true"> </i>
-              </a>&nbsp;&nbsp;&nbsp;&nbsp;
-              <span class="fs-16 fw-light">
-                <strong class="text-light-black">Submission Visibility </strong>
-              </span>
-              <a class="pointer fs-20" (click)="toggleSubmissionVisibility()">
-                <i class="{{ submissionVisibility.icon }}" aria-hidden="true"> </i>
-              </a>
+          <span class="fs-16 fw-light">
+            <strong class="text-light-black">Phase Details </strong>
+          </span>
+          <a class="pointer fs-16" (click)="editPhaseDetails()">
+            <i class="fa fa-pencil" aria-hidden="true"> </i>
+          </a>&nbsp;&nbsp;&nbsp;&nbsp;
+          <span class="fs-16 fw-light">
+            <strong class="text-light-black">Is Public </strong>
+          </span>
+          <a class="pointer fs-20" (click)="togglePhaseVisibility()">
+            <i class="{{ phaseVisibility.icon }}" aria-hidden="true"> </i>
+          </a>&nbsp;&nbsp;&nbsp;&nbsp;
+          <span class="fs-16 fw-light">
+            <strong class="text-light-black">Submission Visibility </strong>
+          </span>
+          <a class="pointer fs-20" (click)="toggleSubmissionVisibility()">
+            <i class="{{ submissionVisibility.icon }}" aria-hidden="true"> </i>
+          </a>
+        </div>
+       </div>
+    </div>
+  </div>
+
+    <!-- Leaderboard Edit Section-->
+    <div class="row settings-section">
+      <div class="col-lg-3 col-md-4 col-sm-3 col-xs-3">
+        <span class="fw-light fs-18">Edit leaderboard details:</span>
+      </div>
+      <br />
+      <div class="ev-action">
+        <div class="col-sm-6 col-xs-12 col-lg-6 col-lr-pad phase-select-box" *ngIf="isChallengeHost">
+          <div class="row margin-bottom-cancel phase-card">
+          <app-selectphase
+            [phases]="filteredPhaseSplits"
+            [phaseSelectionType]="phaseLeaderboardSelectionType"
+            [phaseSelectionListType]="phaseLeaderboardSelectionListType"
+            [phaseSplitSelected]="phaseSplitSelected()"
+            #phasesplitselect
+          >
+          </app-selectphase>
+          </div>
+        <div class="row margin-bottom-cancel phase-card" *ngIf="selectedPhaseSplit">
+          <div class="col s12">
+            <span class="fs-16 fw-light">
+              <strong class="text-light-black">Leaderboard Schema </strong>
+            </span>
+            <a class="pointer fs-16" (click)="editPhaseDetails()">
```

Comment:
[Question] Why are we calling `editPhaseDetails()` function here? As the backend API is not ready so we can add an empty function for now having a `TODO` comment in the controller

Reply:
Done the changes!